### PR TITLE
Fix Issue 19931 - Missing error message when defining postblit, rvalue constructor and copy constructor

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: src/dmd/dsymbolsem.d [hunk 1]
```diff
@@ -473,7 +474,12 @@ private bool buildCopyCtor(StructDeclaration sd, Scope* sc)
         return true;
     }
     else if (cpCtor)
-        return true;
+    {
+        if (hasPostblit)
```
Comment (thewilsonator) [inline src/dmd/dsymbolsem.d]: `return !hasPostblit;`

File: test/fail_compilation/fail19931.d [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19931.d(10): Error: `struct S` may not define both a rvalue constructor and a copy constructor
```
Comment (thewilsonator) [inline test/fail_compilation/fail19931.d]: (not that needs to be addressed in they PR but) that should read "... define both a**n** rvalue constructor..."

**Review by thewilsonator — APPROVED** [review]
Otherwise looks good.

**Review by ghost — APPROVED** [review]

**Comment by RazvanN7** [comment]
@thewilsonator Fixed. Adding auto-merge

**Pull request merged**